### ISSUE TITLE
Black Screen of Death

### ISSUE THREAD

**Issue by Anirban166** [Anirban166/Win-Issues#4]
The infamous black screen on startup problem (encountered after login UI appears, with only the cursor being visible), with multiple possible factors accounting for that. (issues with the most recent windows update or rather bug upgrades, graphics driver issues and what not) 
![IMG_20210402_110840](https://user-images.githubusercontent.com/30123691/113384618-0dc70b00-93a4-11eb-934d-328686253a05.jpg)
Tried every possible option, including the commonplace fixing explorer.exe: (although it was fine)
![IMG_20210402_102841](https://user-images.githubusercontent.com/30123691/113383205-094d2300-93a1-11eb-9708-682c76f9df09.jpg)

Advanced recovery options don't work, thanks to Microsoft being a bitch with their BitLocker recovery scheme when clearly neither I nor did their OS sign me up for this shit in the first place: (wonder when and where did my system get a BitLocker recovery key, or does it even exist?)
![IMG_20210402_102233](https://user-images.githubusercontent.com/30123691/113383394-752f8b80-93a1-11eb-9887-4ae3ab9a4e09.jpg)
![IMG_20210402_002913](https://user-images.githubusercontent.com/30123691/113383639-fb4bd200-93a1-11eb-9a75-3e2a76c2516d.jpg)
![IMG_20210402_002904](https://user-images.githubusercontent.com/30123691/113383649-01da4980-93a2-11eb-97f5-a5f138a0402a.jpg)
![IMG_20210402_103114](https://user-images.githubusercontent.com/30123691/113383712-1f0f1800-93a2-11eb-9450-236cab00126d.jpg)
![IMG_20210402_103445](https://user-images.githubusercontent.com/30123691/113383719-22a29f00-93a2-11eb-89eb-261dc7124d77.jpg)
Using startup settings (for say, booting in safe mode) or the command prompt, Uninstalling the fancy 'quality updates', re-installing OS afresh from the cloud (via wifi with supplied nsk) or locally etc. - none of them work, as apparently one needs to supply this unwarranted key. Needless to say, it was never provided in my account:
<img width="945" alt="Screenshot 2021-04-02 at 11 33 53 AM" src="https://user-images.githubusercontent.com/30123691/113386191-5207da80-93a7-11eb-845a-7bad3b4fd78b.png">

Not to mention:
![IMG_20210402_103331](https://user-images.githubusercontent.com/30123691/113384765-60a0c280-93a4-11eb-881f-79bfe9a12c41.jpg)
So basically Microsoft can fit in some stupid encryption scheme that locks the user out without consent (emplacing a security measure when not asked to any point), but not keep restore points? (yes, system restore is disabled by default)

**Comment by Anirban166** [comment]
Buggy updates turned out to be the root cause. Re-installed OS and disabled auto-update:
- <img height = "20" src = "https://img.shields.io/badge/--black?style=flat&logo=windows" alt = "Windows"> + <kbd>R</kbd>
- services.msc
- Windows update > Startup > Disable 
- Reboot system (`sudo shutdown -r` or <img height = "20" src = "https://img.shields.io/badge/--black?style=flat&logo=windows" alt = "Windows"> + <kbd>X</kbd> + <kbd>U</kbd> + <kbd>R</kbd>)

**Comment by Anirban166** [comment]
Buggy updates turned out to be the root cause. Re-installed OS and disabled auto-update:
- <img height = "20" src = "https://img.shields.io/badge/--black?style=flat&logo=windows" alt = "Windows"> + <kbd>R</kbd>
- services.msc
- Windows update > Startup > Disable 
- <img height = "20" src = "https://img.shields.io/badge/--black?style=flat&logo=windows" alt = "Windows"> + <kbd>X</kbd> + <kbd>U</kbd> + <kbd>R</kbd>

Solved a few days ago.
